beacon_node/network/src/service.rs

```diff
@@ -248,8 +249,15 @@ fn spawn_service<T: BeaconChainTypes>(
                     AttServiceMessage::EnrRemove(subnet_id) => {
                         service.libp2p.swarm.update_enr_subnet(subnet_id, false);
                     }
-                    AttServiceMessage::DiscoverPeers(subnet_id) => {
-                        service.libp2p.swarm.peers_request(subnet_id);
+                    AttServiceMessage::DiscoverPeers(exact_subnet) => {
+                        // add one slot to ensure we keep the peer for the subscription slot
+                        let min_ttl = service.beacon_chain.slot_clock.duration_to_slot_from_genesis(exact_subnet.slot + 1)
```

Comment:
Should this not be `duration_to_slot` rather than `duration_to_slot_from_genesis`?

Reply:
Yea this was part of a workaround to try and use Duration instead of instant to track expiration. But seems like you've already fixed that. Didn't realize I could just skip serialization in PeerInfo for the new field.